backend/DataBase/db_manger.py

```diff
@@ -0,0 +1,102 @@
+import pymysql
+from typing import List
+from sql_queries.queries import *
+from mock_data.transactions import *
+from mock_data.categories import *
+from constants.constans import *
+
+connection = pymysql.connect(
+    host="localhost",
+    user="root",
+    password="",
+    db="bank_app",
+    charset="utf8",
+    cursorclass=pymysql.cursors.DictCursor
+)
+
+if connection.open:
+    print(CHECK_CONNECTION)
+
+
+def get_all_transactions() -> List[dict]:
+    query: str = GET_TRANSACTIONS
+    with connection.cursor() as cursor:
+        cursor.execute(query)
+        result = cursor.fetchall()
+        return result
+
+
+def add_transaction(transaction: Transaction, category: Category) -> None:
+    add_bank_transaction_query: str = ADD_TRANSACTION_TO_TABLE.format(
+        amount=transaction.amount, is_deposit=transaction.is_deposit)
+    add_category_query: str = ADD_CATAGORY_TO_TABLE.format(
+        vendor=category.vendor, category=category.category)
+    add_transaction_category_query: str = ADD_TRANSACTION_CATAGORY_TO_TABLE
+    with connection.cursor() as cursor:
```

Comment:
You can set a type cursor to help you typed to result.
see https://pymysql.readthedocs.io/en/latest/modules/connections.html#pymysql.connections.Connection.cursor****